com.civicrm.hrjobroles/js/src/job-roles/job-roles.constants.js

```diff
@@ -10,11 +10,7 @@ define([
     .constant('settings', {
       classNamePrefix: 'hrjobroles-',
       contactId: decodeURIComponent((new RegExp('[?|&]cid=([^&;]+?)(&|#|;|$)').exec(location.search) || [null, ''])[1].replace(/\+/g, '%20')) || null,
-      debug: true,
-      pathApp: CRM.vars.hrjobroles.path,
-      pathRest: CRM.url('civicrm/ajax/rest'),
-      pathBaseUrl: CRM.vars.hrjobroles.baseURL + '/',
-      pathTpl: 'views/',
-      pathIncludeTpl: 'views/include/'
+      debug: +CRM.debug,
```

Comment:
CRM is a global variable, we usually wrap the whole thing into a function and then pass the variable. I am not saying I am a huge fan of doing this, but that would be consistent.

```js
/* eslint-env amd */

(function (CRM) {
  define([
    // ...
  ], function (/* ... */) {
    // ...
  });
}(CRM));

```